SEED-platform/seed
# Notes Improvements

#### What's this PR do?
- Adds a link to the inventory list view for each record with a link to the notes.  It appears blue and shows the number of notes if any exist
- When unmerging a record that contains notes, the notes are duplicated to each new record with the original timestamps preserved.
- When merging records that contain notes, the notes are combined into the new record with the original timestamps preserved.  If exact note duplicates exist (for instance, from unmerging a record prior to re-merging) the exact duplicates are de-duplicated.
- Small ui tweaks like disabling the `save note` button if the contents are empty to avoid 400 responses
- Also fixes a small issue on the inventory list view where selecting the `and` `or` `exlude` buttons weren't acting as radio buttons
#### How should this be manually tested?
Create some notes for properties/taxlots, test merging and unmerging those records, check that notes never get deleted, that timestamps don't change, and that exact notes get deduplicated.
#### Screenshots
![image](https://user-images.githubusercontent.com/411466/35464430-1fbaa298-02b3-11e8-8e9c-95b8dffcf7f2.png)
